hera_mc/cm_utils.py

```diff
@@ -658,6 +665,20 @@ def csv_table(headers, table):
     return s_table
 
 
+def table_table(headers, table_data, output_format=None):
```

Comment:
That's a pretty confusing function name.

Reply:
Yes indeed.  I've renamed general_table_handler